pkg/plugin/backup_pvc_action_plugin.go

```diff
@@ -24,22 +24,15 @@ type NewPVCBackupItemAction struct {
 
 // AppliesTo returns information indicating that the PVCBackupItemAction should be invoked to backup PVCs.
 func (p *NewPVCBackupItemAction) AppliesTo() (velero.ResourceSelector, error) {
-	p.Log.Info("PVCBackupItemAction AppliesTo for vSphere")
+	p.Log.Info("VSphere PVCBackupItemAction AppliesTo")
```

Comment:
The logging seems incomplete.

Reply:
This line of log is just used for logging the `AppliesTo` func gets called. We can improve it later if necessary.